test/e2e/v4/plugin_cluster_test.go

```diff
@@ -107,6 +107,14 @@ func Run(kbc *utils.TestContext) {
 	err = kbc.Tidy()
 	ExpectWithOffset(1, err).NotTo(HaveOccurred())
 
+	By("run make manifests")
+	err = kbc.Make("manifests")
+	ExpectWithOffset(1, err).NotTo(HaveOccurred())
+
+	By("run make generate")
+	err = kbc.Make("generate")
```

Comment:
@camilamacedo86 Thanks for taking care of CI - I've rebased onto master to pickup those fixes, though I _thought_ that I already took care of the e2e tests with this change. Am I missing something?

Reply:
I added the tests: https://github.com/kubernetes-sigs/kubebuilder/pull/3468 and I could not face the issue.
So, I am not sure what is the problem with the default scaffold that we have. 

Could you please raise an PR with only the new test which should be failing since this PR is not merged for we ensure the fix?

Reply:
Hi @mjlshen 

See that the PR is merged and the targets are not failing.
So, when the error is faced? Only when we run `kustomize build config/crd` direct?

Can you please clarify when/how to reproduce the issue?

Reply:
I ended up reproducing this in #3472 and pulling those changes into this PR